### PR TITLE
Mixtral chatbot

### PR DESCRIPTION
**Pull request by sophiamyang**

### CODE REVIEW

File: posts/mixtral/images/chatbot.png
Comment (maximlt) [inline posts/mixtral/images/chatbot.png:1]: Could you reduce the size of the image? 1MB is a bit too big for a thumbnail. The thumbnails in the blog are resized to have a height of 200px, so you could downsize it so it has that height. And I usually compress PNGs using https://tinypng.com/.

File: posts/mixtral/index.ipynb [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+    "- Defines the model `mistralai/Mixtral-8x7B-Instruct-v0.1`\n",
+    "- Uses the Tokenizer from this model to format the input user message\n",
+    "- Uses the transformers pipeline to specify the text-generation pipeline, the model, and the 4-bit quantization.\n",
+    "- Uses the `TextStreamer` to stream the text output \n",
+    "\n",
+    "```python\n",
+    "# REF: Code adapted from https://huggingface.co/blog/mixtral\n",
+    "!pip install -U \"transformers==4.36.0\"  --upgrade\n",
+    "!pip install accelerate\n",
+    "!pip install bitsandbytes\n",
```
Comment (maximlt) [inline posts/mixtral/index.ipynb]: Maybe replace these commands by some text saying these packages need to be installed? Not sure it's good advice to have users run those in their environment (you're running it in Colab maybe?).

**Review by maximlt — COMMENTED** [review]
I'd be nice to add a snapshot of the Python environment you used to run the code in the `/repro` folder, running e.g. `pip freeze > requirements.txt`, `conda env export > environment.yml`. This is for us to be able to reproduce the content of the blog in the future, just in case.

File: posts/mixtral/index.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
+---
+title: "Build a Mixtral Chatbot with Panel"
+date: "2023-12-12"
+description: "With Transformers and llama.cpp"
+author:
+  - Andrew Huang
+  - Philipp Rudiger
+  - Sophia Yang
+categories: [showcase, panel]
```
Comment (maximlt) [inline posts/mixtral/index.md:9]: You could add a new category like `ai`, `llm`, `chatbot`, etc.

File: posts/mixtral/index.md [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+
+
+If you are interested in learning more about how to build AI chatbot in Panel, please read our related blog posts: 
+
+- [Building AI Chatbots with Mistral and Llama2](https://medium.com/@sophiamyang/building-ai-chatbots-with-mistral-and-llama2-9c0f5abc296c) 
+- [Building a Retrieval Augmented Generation Chatbot](https://medium.com/@sophiamyang/building-a-retrieval-augmented-generation-chatbot-d567a24fcd14)
+- [How to Build Your Own Panel AI Chatbots](https://medium.com/@sophiamyang/how-to-build-your-own-panel-ai-chatbots-ef764f7f114e)
+- [Build a RAG chatbot to answer questions about Python libraries](https://blog.holoviz.org/posts/fleet_ai/)
+
+If you find Panel useful, please consider giving us a star on Github ([https://github.com/holoviz/panel](https://github.com/holoviz/panel)). Happy coding!
```
Comment (maximlt) [inline posts/mixtral/index.md:221]: ❤️ 🌟

File: posts/mixtral/images/chatbot.png
Comment (sophiamyang) [reply posts/mixtral/images/chatbot.png:1]: resized the image, not sure how to change the height though

File: posts/mixtral/index.md (hunk 2 above)
Comment (sophiamyang) [reply posts/mixtral/index.md]: good idea thanks!

File: posts/mixtral/index.ipynb (hunk 1 above)
Comment (sophiamyang) [reply posts/mixtral/index.ipynb]: hmmm so I took the code from the HF blog and they used !pip xxx...

**Pull request merged**